source/fuzz/transformation_add_global_variable.cpp

```diff
@@ -93,15 +93,13 @@ bool TransformationAddGlobalVariable::IsApplicable(
 void TransformationAddGlobalVariable::Apply(
     opt::IRContext* ir_context,
     TransformationContext* transformation_context) const {
-  fuzzerutil::AddGlobalVariable(
+  opt::Instruction* new_instruction = fuzzerutil::AddGlobalVariable(
```

Comment:
Since we've already removed all `fuzzerutil::AddType*` functions, I wonder whether we need to keep `fuzzerutil::Add*Variable`.

Reply:
We do - or at least it's not as easy to get rid of them - because they're used in a few places. The AddType ones were trivial to get rid of. (The one for adding function types wasn't, though, so I left it.)

Reply:
I think that's OK to have them then since we wouldn't want to duplicate the code.